plugins/web/opentelemetry-instrumentation-web-exception/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@opentelemetry/instrumentation-web-exception",
+  "version": "0.0.1",
+  "description": "OpenTelemetry instrumentation for web exceptions",
+  "main": "build/src/index.js",
+  "module": "build/esm/index.js",
+  "esnext": "build/esnext/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "lint:readme": "node ../../../scripts/lint-readme.js",
+    "prewatch": "npm run precompile",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "prepublishOnly": "npm run compile",
+    "tdd": "wtr --watch",
+    "test:browser": "wtr --coverage",
+    "watch": "tsc --build -watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json"
+  },
+  "keywords": [
+    "opentelemetry",
+    "exception",
+    "web",
+    "tracing",
+    "plugin",
+    "browser",
+    "error"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": "^18.19.0 || >=20.6.0"
+  },
+  "files": [
+    "build/esm/**/*.js",
+    "build/esm/**/*.map",
+    "build/esm/**/*.d.ts",
+    "build/esnext/**/*.js",
+    "build/esnext/**/*.map",
+    "build/esnext/**/*.d.ts",
+    "build/src/**/*.js",
+    "build/src/**/*.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.3.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.22.17",
+    "@jsdevtools/coverage-istanbul-loader": "3.0.5",
+    "@opentelemetry/api": "^1.3.0",
+    "@opentelemetry/api-logs": "^0.57.2",
+    "@opentelemetry/sdk-events": "^0.57.2",
+    "@opentelemetry/sdk-logs": "^0.57.2",
+    "@opentelemetry/sdk-trace-base": "^1.0.0",
+    "@rollup/plugin-commonjs": "^26.0.0",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@types/chai": "^4.3.10",
+    "@types/mocha": "10.0.10",
+    "@types/node": "18.18.14",
+    "@types/sinon": "17.0.4",
+    "@web/dev-server-esbuild": "^1.0.1",
+    "@web/dev-server-rollup": "^0.6.1",
+    "@web/test-runner": "^0.18.0",
+    "chai": "^4.3.10",
+    "sinon": "15.2.0",
+    "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "@opentelemetry/api-events": "^0.57.2",
```

Comment:
The Events API and SDK have been removed. This PR should be updated to use just the Log API.